lib/spack/spack/environment/environment.py

```diff
@@ -102,6 +102,16 @@
 default_view_link = 'all'
 
 
+def get_installed_package_specs():
```

Comment:
Nitpick. We can avoid `get_` and possibly `package` in the name:
```suggestion
def installed_specs():
```
When a client calls this function I think:
```python
spack.environment.installed_specs()
```
will still make sense. What do you think?

Reply:
It does have a kind of symmetry with `uninstalled_specs`.  My concern was there is an `installed_specs` in `cmd/common/arguments.py` and it is used in a number of contexts as a variable but I will assume that won't cause problems.